notebooks-d2l/finetune_cnn_jax.ipynb

```diff
@@ -0,0 +1,816 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        x = nn.BatchNorm(use_running_average=True)(x)
</pre><p>Should there be a train/test flag to decide if <code>use_running_average</code> is True or False? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/probml-notebooks/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>